docs/tutorial.rst

```diff
@@ -93,9 +93,9 @@ Now that you have the destination branch in origin go ahead and run SCVersioning
 
 .. code-block:: bash
 
-    sphinx-versioning -r feature_branch push docs gh-pages .
+    sphinx-versioning push -r feature_branch push docs gh-pages .
```

Comment:
Same, remove redundant "push". Basically you're moving "-r feature_branch" to the right.